src/index.ts

```diff
@@ -59,21 +59,11 @@ const {
 
 export const MTS_SUPPORTED_NODE_VERSION = 16
 export const LOADER_SUPPORTED_NODE_VERSION = 20
-export const STRIP_TYPES_DEFAULT_NODE_VERSION = 23
-export const STRIP_TYPES_SUPPORTED_NODE_VERSION = 22
+export const TYPESCRIPT_DEFAULT_NODE_VERSION = 23.6
+export const TYPESCRIPT_SUPPORTED_NODE_VERSION = 22.1
```

Comment:
Should it be `22.10`, then we can not use `Number.parseFloat` in this case.

cc @nwidynski